flink-runtime/src/main/java/org/apache/flink/runtime/operators/chaining/ChainedDriver.java

```diff
@@ -63,8 +64,8 @@
 
 	
 	public void setup(TaskConfig config, String taskName, Collector<OT> outputCollector,
-			AbstractInvokable parent, ClassLoader userCodeClassLoader, ExecutionConfig executionConfig,
-			Map<String, Accumulator<?,?>> accumulatorMap)
+					  AbstractInvokable parent, UserCodeClassLoader userCodeClassLoader, ExecutionConfig executionConfig,
```

Comment:
Note: We need to forward the `UserCodeClassLoader`, so that we can pass it into the `DistributedRuntimeUDFContext`. 
But I was able reduce the `userCodeClassLoader` field to `ClassLoader` again.